storage-resize-images/functions/src/index.ts

```diff
@@ -144,6 +149,15 @@ export const generateResizedImage = functions.storage.object().onFinalize(
       });
 
       const results = await Promise.all(tasks);
+      console.log("filePath=" + filePath)
+      console.log("object.name=" + object.name)
+      eventChannel && eventChannel.publish({
```

Comment:
Heads up - I think we need to await eventChannel.publish since it returns a promise. If we don't, errors will get lost in the ether & won't be logged.